README.md

```diff
@@ -72,13 +72,18 @@ git checkout asf-site
 git pull
 # create a branch for the publishing
 git checkout -b publish_blog
-# push code upstream
-git push 
 # copy content built from _site directory
 cp -R ../datafusion-site/_site/* .
 git commit -a -m 'Publish blog content'
+# push code upstream
+git push 
 ```
 
 #### Make PR, targeting the `asf-site` branch
 For example, see https://github.com/apache/datafusion-site/pull/9
 
+#### Check site status
+
+The website is updated from the `asf-site` branch on a schedule, so it may take
+some time for the  changes to be reflected. You can check the status at 
```

Comment:
I think I was wrong about the schedule. It seems to check on a schedule but also responds to pushes.

Reply:
Yes, I see now it was a temporary bug. I will revise the wording